modules/py_csi.c

```diff
@@ -131,7 +131,7 @@ static mp_obj_t py_omv_csi_shutdown(mp_obj_t enable) {
 static MP_DEFINE_CONST_FUN_OBJ_1(py_omv_csi_shutdown_obj, py_omv_csi_shutdown);
 
 static mp_obj_t py_omv_csi_flush() {
-    framebuffer_update_jpeg_buffer();
+    framebuffer_update_jpeg_buffer(csi.fb);
```

Comment:
Shouldn't this use `framebuffer_t *fb = framebuffer_get(0);` so you can specify the frame buffer to flush later on?

Reply:
No. `framebuffer_get(0);` is for when you need a frame buffer and don't have access to a CSI instance, for example `py_fir`, `py_tof`. This function will eventually receive a CSI ID.